metadata.md

```diff
@@ -36,13 +37,13 @@ Results for a measurement of a single property of SARS-CoV-2 wastewater test, fo
 -   **analysisDate**: Date the data was analysed in the lab.
 
 -   **reportDate**: Data the data was reported. One sampleID may have updated reports based on updates to assay method or reporting standard. In this situation, use the original `sampleID` but updated `measurementID`, `reportDate` and `assayID` (if needed).
-
+ - **childSampleID**: If this is a sample with many smaller samples either because of pooling or sub-sampling this indicates a coma seperated list of child samples.
```

Comment:
Just noting that this is maybe the first time we include a list? I think OK if the list is small. However, we've generally used a table. 

I expect this to be not used too often and also a short list.